plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -517,6 +532,25 @@ private class DokkaDescriptorVisitor( // TODO: close this class and make it priv
         )
     }.let(::Annotations)
 
+    fun DeclarationDescriptor.convertType(dri: DRI, context: DokkaContext): Bound = when (val desc = this) {
+        is ValueParameterDescriptor -> desc.type
+        is FunctionDescriptor -> desc.returnType
+        is PropertyDescriptor -> desc.returnType
```

Comment:
This is a duplicated comment, but I feel that one of this places can be incorrect.